### ISSUE TITLE
Doesn't work inside a table cell

### ISSUE THREAD

**Issue by joshualawson** [ALegendsTale/obsidian-color-palette#17]
Doesn't render the color palette inside a table cell:

Example:
```
| col 1 | col 2 | col 3 |
|---|---|---|
| test | col2 test | ```palette #ffffff #000```|

```

**Comment by ALegendsTale** [comment]
Yes, unfortunately placing color palettes inside of tables is not supported and I'm thinking it's probably out of scope for this project.

My best recommendation is to try [obsidian-css-inlay-colors](https://github.com/GRA0007/obsidian-css-inlay-colors) which is a lovely plugin created by GRA0007.

Using it is simple and single colors can be defined with a hex code inside of a code block (e.g. ``` `#000` ```). It will work inside of tables and your request could be implemented like so:
```
| col 1 | col 2 | col 3 |
|---|---|---|
| test | col2 test | `#ffffff` `#000`|
```
